gramex/handlers/proxyhandler.py

```diff
@@ -7,10 +7,12 @@
 from gramex.transforms import build_transform
 from gramex.config import app_log
 from gramex.http import MOVED_PERMANENTLY, FOUND
-from .basehandler import BaseHandler
+from .basehandler import BaseWebSocketHandler, BaseHandler
+from tornado.websocket import WebSocketHandler
+from gramex.handlers import WebSocketHandler as GWebSocketHandler
```

Comment:
Can we rename GWebSocketHandler to WebsocketHandler and import Tornado's WebSocketHandler as TornadoWebSocketHandler?